### DISCUSSION TITLE
Synthesized Speech with noise

### DISCUSSION THREAD

**Discussion by jmlcoliveira** [coqui-ai/TTS#2452] [General Q&A]
Hey. 
So I have fine tunned my model using the tts_models/en/ljspeech/glow-tts.
The generated speech if I use the Grifin-Lin vocoder, sounds a bit robotic, but I can understand what is been said.
This is how it sounds:

https://user-images.githubusercontent.com/27073348/227272662-1b767c64-d834-432c-8233-2e9342ba01ce.mp4

I tried to use other vocoders, but it is not producing any understandable sound. The ones I tried were this:
    "vocoder_models/en/ljspeech/hifigan_v2"
    "vocoder_models/en/ljspeech/multiband-melgan"
    "vocoder_models/en/ljspeech/univnet"
    "vocoder_models/universal/libri-tts/fullband-melgan"
Using multiband-melgan vocoder for examples sounds like this.
https://user-images.githubusercontent.com/27073348/227272882-924c4643-d11e-43a2-80d3-e5b97592052c.mp4

Any advices for me?

**Comment by p0p4k** [comment]
Tune some hyperparameters. Add a few layers here and there (by increasing num_layers of that block). Might help. Good luck!
